File: Neo4j/neo4j_nbp/Frontend/PolaroIT/src/app/models/korisnik.ts
import { Time } from '@angular/common';

export interface Korisnik {
  ime: string;
  prezime: string;
  rodjedan: string;
  korisnicko_ime: string;
  lozinka: string;
  email: string;
  online: boolean;
  drzava: string;
  profilnaSlika:string;
}

File: Neo4j/neo4j_nbp/Backend/Neo4J/Models/Korisnik.cs
using Microsoft.Extensions.Options;
using System.Collections.Generic;

namespace Neo4J.Models
{
    public class Korisnik
    {
        // Node
        public string ime { get; set; }

        public string prezime { get; set; }

        public string rodjedan { get; set; }

        public string korisnicko_ime { get; set; }

        public string lozinka { get; set; }

        public string email { get; set; }

        public string profilnaSlika { get; set; }

        public TipKorsnika tip { get; set; }

       public Drzava drzava { get; set; }

       public List<Slika> galerija { get; set; }

       public Preference preference { get; set; }
       public Following pracenje { get; set; }
       public bool online { get; set; }   
    }
}
